deploy/chart/templates/0000_50_olm_02-services.yaml

```diff
@@ -31,9 +31,48 @@ spec:
   type: ClusterIP
   ports:
   - name: https-metrics
-    port: 8081
+    port: {{ .Values.catalog.service.internalPort }}
     protocol: TCP
-    targetPort: metrics
+    targetPort: {{ .Values.catalog.service.externalPort }}
+  selector:
+    app: catalog-operator
+{{ end }}
+{{ if .Values.pprof.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: olm-operator-pprof
+  namespace: {{ .Values.namespace }}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: olm-operator-serving-cert
+  labels:
+    app: olm-operator
+spec:
+  type: ClusterIP
+  ports:
+  - name: https-pprof
+    port: {{ .Values.olm.service.internalPort }}
+    protocol: TCP
+    targetPort: {{ .Values.olm.service.externalPort }}
+  selector:
+    app: olm-operator
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: catalog-operator-pprof
+  namespace: {{ .Values.namespace }}
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: catalog-operator-serving-cert
+  labels:
+    app: catalog-operator
+spec:
+  type: ClusterIP
+  ports:
+  - name: https-pprof
+    port: {{ .Values.catalog.service.internalPort }}
+    protocol: TCP
+    targetPort: {{ .Values.catalog.service.externalPort }}
   selector:
     app: catalog-operator
 {{ end }}
```

Comment:
These should live in OCP, since they rely on service-ca-operator.

Reply:
Good point. I'll remove these here.